core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -487,9 +487,9 @@ namespace kagome::consensus::grandpa {
     SL_DEBUG(logger_,
              "{} has received from {}: "
              "voter_set_id={} round={} for block #{} hash={}",
-             msg.vote.is<Prevote>()
-                 ? "Prevote"
-                 : msg.vote.is<Precommit>() ? "Precommit" : "PrimaryPropose",
+             msg.vote.is<Prevote>()     ? "Prevote"
```

Comment:
clang-format off?

Reply:
It was made by clang-format-12. It looks better, IMHO.